test/Calibur.execute.t.sol

```diff
@@ -319,7 +403,47 @@ contract CaliburExecuteTest is TokenHandler, HookHandler, ExecuteFixtures, Deleg
         uint256 nonce = key << 64 | seq;
 
         // Create hash of the calls + nonce and sign it
-        BatchedCall memory batchedCall = CallUtils.initBatchedCall().withCalls(calls).withShouldRevert(true);
+        BatchedCall memory batchedCall = CallUtils.initBatchedCall().withCalls(calls).withRevertOnFailure(true);
+        SignedBatchedCall memory signedBatchedCall = CallUtils.initSignedBatchedCall().withBatchedCall(batchedCall)
+            .withNonce(nonce).withKeyHash(p256Key.toKeyHash());
+        bytes32 hashToSign = signerAccount.hashTypedData(signedBatchedCall.hash());
+        bytes memory signature = p256Key.sign(hashToSign);
+
+        bytes memory revertData = bytes("revert");
+        mockExecutionHook.setBeforeExecuteRevertData(revertData);
+        Settings keySettings = SettingsBuilder.init().fromHook(mockExecutionHook);
+
+        vm.prank(address(signerAccount));
+        signerAccount.update(p256Key.toKeyHash(), keySettings);
+
+        // Expect the call to revert
+        bytes memory wrappedSignature = abi.encode(signature, EMPTY_HOOK_DATA);
+        vm.expectRevert("revert");
+        signerAccount.execute(signedBatchedCall, wrappedSignature);
+
+        // Unset the hook revert
+        mockExecutionHook.setBeforeExecuteRevertData(bytes(""));
+
+        signerAccount.execute(signedBatchedCall, wrappedSignature);
+        assertEq(tokenA.balanceOf(address(receiver)), 1e18);
+    }
+
+    function test_execute_batch_opData_revertOnFailureFalse_withHook_beforeExecute() public {
```

Comment:
no opData

Reply:
nit maybe rename to

test_execute_withSignature_revertOnFailureFalse_withHook_beforeExecute_reverts

to signal we DO expect a revert